packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card-back.tsx

```diff
@@ -1,52 +1,67 @@
-import { Icon, Text } from '@deriv/components';
 import classNames from 'classnames';
+import { Icon, Text } from '@deriv/components';
 import { jurisdictionVerificationContents } from 'Constants/jurisdiction-contents/jurisdiction-verification-contents';
 import { TJurisdictionCardBackProps } from 'Containers/props.types';
 import React from 'react';
 
-const JurisdictionCardBack = ({ card_classname, toggleCardFlip, verification_docs }: TJurisdictionCardBackProps) => (
+const JurisdictionCardBack = ({
+    card_classname,
+    toggleCardFlip,
+    verification_docs,
+    is_card_selected,
+}: TJurisdictionCardBackProps) => (
     <div
-        className={classNames(`${card_classname}__card-content-container`, `${card_classname}__card-flipped-container`)}
+        className={classNames(card_classname, 'cfd-card-back', {
+            [`${card_classname}--selected selected-card`]: is_card_selected,
```

Comment:
why do we add 2 classes here?
${card_classname}--selected and selected-card

Reply:
@amina-deriv css :not selector was not working with the scss based names. so i wrote a new class for not selector alone.
`&:not(.selected-card):hover {`
Is there a way to resolve already existing classname `&--selected` inside :not operator like `&:not(.&--selected):hover {`?